spyder/plugins/completion/kite/widgets/calltoaction.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+from qtpy.QtCore import Qt, QTimer, QUrl
+from qtpy.QtGui import QDesktopServices
+from qtpy.QtWidgets import (QLabel, QApplication,
+                            QVBoxLayout, QFrame, QHBoxLayout,
+                            QPushButton)
+from spyder.utils.syntaxhighlighters import get_color_scheme
+
+from spyder.config.base import _
+from spyder.config.gui import get_font
+from spyder.config.manager import CONF
+from spyder.plugins.completion.kite.bloomfilter import KiteBloomFilter
+from spyder.plugins.completion.kite.parsing import find_returning_function_path
+from spyder.plugins.completion.kite.utils.status import check_if_kite_installed
+from spyder.plugins.completion.fallback.actor import FALLBACK_COMPLETION
+
+COVERAGE_MESSAGE = (
+    _("No completions found."
+      " Get completions for this case and more by installing Kite.")
+)
+
+
+class KiteCallToAction(QFrame):
+    def __init__(self, textedit, ancestor):
+        super(KiteCallToAction, self).__init__(ancestor)
+        self.textedit = textedit
+
+        self.setFrameStyle(54)
+        self.setAutoFillBackground(True)
+        self.setWindowFlags(Qt.SubWindow | Qt.FramelessWindowHint)
+        self.setFocusPolicy(Qt.NoFocus)
```

Comment:
Please, below this line, add the following to have the correct behavior of the border when using the dark theme:
```python
    if is_dark_interface():
            self.setObjectName("kite-call-to-action")
            self.setStyleSheet(
                self.styleSheet() + ("#kite-call-to-action "
                                     "{ border: 1px solid; "
                                     "  border-color: #32414B; "
                                     "  border-radius: 4px;} "
                                     "#kite-call-to-action:hover "
                                     "{ border:1px solid #148CD2; }"))
```

You will need to add the import for `is_dark_interface` at the imports section of the file. The import is 

```python
from spyder.utils.icon_manager import is_dark_interface
```

Reply:
lovely, this fixes it, thanks!